daemon/settings.h

```diff
@@ -71,6 +71,14 @@ typedef enum
     // keep app_agreed_name[] in sync
 } app_agreed_t;
 
+typedef enum {
+    APP_GRANT_DEFAULT, // Take default
+    APP_GRANT_ALWAYS,  // Always allow all permissions
+    APP_GRANT_LAUNCH,  // Allow launching, user may control permissions
```

Comment:
One day we will need an ability for MDM to override user's selections. I.e. MDM would be able to lock some permissions, whether in enabled or disabled state, so a user will not be able to override it. The same could happen with the primary device user overriding permissions for secondary users. But I'm not sure, how will it be implemented.